client/src/actions/index.js

```diff
@@ -23,52 +23,46 @@ function obsAnnotationFetchAndLoad(dispatch, schema) {
 
   const plimit = new PromiseLimit(5);
   return Promise.all(
-    columns.map((col) =>
-      plimit.add(() => {
-        const path = `annotations/obs?annotation-name=${encodeURIComponent(
-          col.name
-        )}`;
-        const url = `${globals.API.prefix}${globals.API.version}${path}`;
-        return doBinaryRequest(url).then((buffer) => {
-          const df = Universe.matrixFBSToDataframe(buffer);
-          dispatch({
-            type: "universe: column load success",
-            dim: "obsAnnotations",
-            dataframe: df,
-          });
-        });
-      })
-    )
+    columns
+      .filter(col => col.name !== index)
```

Comment:
why is this being added back to the code?  It is done on line 20, so this is redundant.

Reply:
Good catch. Likely got added upon rebase.